package.json

```diff
@@ -4,63 +4,63 @@
   "private": true,
   "devDependencies": {
     "autoprefixer": "6.4.1",
-    "babel-cli": "6.24.1",
-    "babel-core": "6.14.0",
+    "babel-cli": "6.26.0",
+    "babel-core": "6.26.3",
     "babel-eslint": "6.1.2",
-    "babel-jest": "19.0.0",
+    "babel-jest": "22.4.4",
     "babel-loader": "6.2.5",
-    "babel-plugin-transform-class-properties": "6.11.5",
-    "babel-plugin-transform-object-rest-spread": "6.8.0",
-    "babel-plugin-transform-react-constant-elements": "6.9.1",
-    "babel-plugin-transform-regenerator": "6.14.0",
-    "babel-plugin-transform-runtime": "6.15.0",
-    "babel-preset-env": "1.6.0",
+    "babel-plugin-transform-class-properties": "6.24.1",
+    "babel-plugin-transform-object-rest-spread": "6.26.0",
+    "babel-plugin-transform-react-constant-elements": "6.23.0",
+    "babel-plugin-transform-regenerator": "6.26.0",
+    "babel-plugin-transform-runtime": "6.23.0",
+    "babel-preset-env": "1.7.0",
     "babel-preset-flow": "6.23.0",
-    "babel-preset-latest": "6.14.0",
-    "babel-preset-react": "6.11.1",
-    "babel-runtime": "6.11.6",
+    "babel-preset-latest": "6.24.1",
+    "babel-preset-react": "6.24.1",
+    "babel-runtime": "6.26.0",
     "blob-util": "1.2.1",
-    "bootstrap": "3.3.7",
+    "bootstrap": "4.1.1",
     "case-sensitive-paths-webpack-plugin": "1.1.4",
     "chalk": "1.1.3",
-    "connect-history-api-fallback": "1.3.0",
+    "connect-history-api-fallback": "1.5.0",
     "copy-webpack-plugin": "3.0.1",
     "cross-spawn": "4.0.0",
     "css-loader": "0.24.0",
-    "detect-port": "1.0.0",
+    "detect-port": "1.2.3",
     "eslint": "3.8.1",
     "eslint-config-standard": "~6.2.1",
     "eslint-config-standard-jsx": "~3.2.0",
     "eslint-config-standard-react": "~4.2.0",
     "eslint-loader": "1.5.0",
-    "eslint-plugin-flowtype": "2.32.1",
-    "eslint-plugin-import": "2.7.0",
-    "eslint-plugin-promise": "~3.3.0",
+    "eslint-plugin-flowtype": "2.46.3",
+    "eslint-plugin-import": "2.12.0",
+    "eslint-plugin-promise": "~3.7.0",
```

Comment:
I'm not sure: do we want to avoid version ranges completely or `~` tildas are ok?

Reply:
sure, I missed that while concentrating on the rest. Removing semver signs